platforms/corda-ent/charts/idman/values.yaml

```diff
@@ -71,84 +61,117 @@ vault:
   # Eg. serviceaccountname: vault-auth
-  serviceaccountname:
-  # Provide the vault path where the  certificates are stored
+  serviceAccountName:
+  # Provide the vault path where the certificates are stored
   # Eg. certsecretprefix: secret/cenm-org-name
-  certsecretprefix:
+  certSecretPrefix:
  
+#############################################################
+#                    Idman Configuration                    #
+#############################################################
 
-# Provide volume related specifications
-volume:
- # Ex baseDir: /opt/corda
-  baseDir: 
-
-# idman service 
+# Idman 'main' service
 service:
-  # Ex. port: 10000
+  # Eg. port: 10000
   port: 
 
+# Internal service, inside the K8s cluster
 serviceInternal:
-  # internal port inside cluster
-  # Ex. port: 5052
+  # Eg. port: 5052
   port: 
 
+# Certificate Revocation Authority (CRA) service, inside the K8s cluster
 serviceRevocation:
-  # Revocation port inside cluster
-  # Ex port: 5053
+  # Eg. port: 5053
   port: 
 
-# ssh service related configuration
+# Provide the service details for setting up Shell access to the service.
 shell:
-  # Ex. sshdPort: 2222
+  # Te port to use for the SSH service
+  # Eg. sshdPort: 2222
   sshdPort: 
-  # ssh username
-  # Ex. user: idman
+  # SSH username
+  # Eg. user: idman
   user: 
-  # ssh password
-  # Ex. idmanP
+  # SSH password
+  # Eg. password:"idmanP
   password:
 
+#############################################################
+#                Database Options and Configuration         #
+#############################################################
 # Database configuration
 database:
-  # Ex. driverClassName: "org.h2.Driver"
+  # Java class name to use for the database
+  # Eg. driverClassName: "org.h2.Driver"
   driverClassName:
-  # Ex. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
+  # The DB connection URL 
+  # Eg. url: "jdbc:h2:file:./h2/identity-manager-persistence;DB_CLOSE_ON_EXIT=FALSE;LOCK_TIMEOUT=10000;WRITE_DELAY=0;AUTO_SERVER_PORT=0"
   url: 
-  # Ex. user: "example-db-user"
-  user: 
-  # Ex. password: "example-db-password"
+  # DB user
+  # Eg. user: "example-db-user"
+  user:
+  # DB password 
+  # Eg. password: "example-db-password"
   password: 
-  # Ex. runMigration: "true"
+  # Migrations of database can be run as part of the startup of Idman, if set to true. 
+  # If set to false, it will be run prior to setting up the Idman.
+  # Eg. runMigration: "true"
   runMigration: 
 
-# Specify the maximum size of the memory allocation pool
-# cordaJarMx: 1
-cordaJarMx: 
-
-# healthCheckNodePort set to get rid of logs pollution
-# Ex. healthCheckNodePort: 0
-healthCheckNodePort: 
+#############################################################
+#                       Settings                            #
+#############################################################
+# Provide volume related specifications
```

Comment:
volume till before healthcheck should be under a common key.

Reply:
Fixed in [ae9a768](https://github.com/hyperledger-labs/blockchain-automation-framework/pull/992/commits/ae9a76899ddcce9ea9e2067c3ba79ebc784c08c5); all values have been grouped under common key `config`